modules/apps/adaptive-media/adaptive-media-image-service/bnd.bnd

```diff
@@ -1,6 +1,6 @@
 Bundle-Name: Liferay Adaptive Media Image Service
 Bundle-SymbolicName: com.liferay.adaptive.media.image.service
 Bundle-Version: 5.0.35
-Liferay-Require-SchemaVersion: 2.2.0
+Liferay-Require-SchemaVersion: 2.3.0
```

Comment:
@julschong I do not think you need to do a minor version change, a micro version change should be enough as there is actually no database table change at all.

Reply:
Ok, let me change it